src/plugins/paginate/paginate-en.hbs

```diff
@@ -0,0 +1,1060 @@
+				<h3 class="mrgn-tp-0">Impedit natus aperiam nemo adipisci?</h3>
+				<p>Illum dolore ipsam corporis atque quisquam cupiditate quod aliquid quaerat, saepe architecto aspernatur ratione autem!</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Assumenda dignissimos beatae doloremque dolores.</h3>
+				<p>Aliquid temporibus sunt quis illum, voluptatem atque. Blanditiis voluptatum iure cupiditate recusandae hic sint aperiam.</p>
+			</div>
+		</article>
+	</div>
+	<details>
+		<summary>Source code</summary>
+		<pre><code>&lt;div class="wb-paginate provisional row wb-eqht-grd" data-wb-paginate='{
+		"itemsPerPage": 12
+	}'>
+	&lt;article class="col-sm-6 col-md-4 col-lg-3">
+		&lt;div class="well">
+			&lt;h3 class="mrgn-tp-0">Lorem ipsum dolor sit amet.&lt;/h3>
+			&lt;p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Labore sequi eius molestiae, non rerum officia?&lt;/p>
+		&lt;/div>
+	&lt;/article>
+	&lt;article class="col-sm-6 col-md-4 col-lg-3">
+		&lt;div class="well">
+			&lt;h3 class="mrgn-tp-0">Unde ullam aliquid numquam laudantium!&lt;/h3>
+			&lt;p>Facere modi possimus a hic voluptatum reiciendis, quod laboriosam, dolor voluptate praesentium alias inventore eveniet.&lt;/p>
+		&lt;/div>
+	&lt;/article>
+	&lt;article class="col-sm-6 col-md-4 col-lg-3">
+		&lt;div class="well">
+			&lt;h3 class="mrgn-tp-0">Corporis repellat nostrum inventore excepturi?&lt;/h3>
+			&lt;p>Consequuntur nostrum suscipit nisi temporibus facere, explicabo inventore reiciendis maxime. Repudiandae consequuntur eligendi quod recusandae!&lt;/p>
+		&lt;/div>
+	&lt;/article>
+	&lt;article class="col-sm-6 col-md-4 col-lg-3">
+		&lt;div class="well">
+			&lt;h3 class="mrgn-tp-0">Voluptate explicabo architecto iste a?&lt;/h3>
+			&lt;p>Dolores minus, provident nihil laudantium dicta nam, rerum doloremque ab nesciunt, vitae optio excepturi voluptas.&lt;/p>
+		&lt;/div>
+	&lt;/article>
+	...
+&lt;/div></code></pre>
+	</details>
+</section>
+
+<section id="filteredEventsList">
+	<h2>Example 4 - Events list</h2>
+	<p>This example leverages the following options: <code>itemsPerPage</code>, <code>section</code>, <code>selector</code>, and <code>uiTarget</code></p>
+	<p>This example also leverages the following other two plugins: <b>Filter</b> and <b>Tagfilter</b>. In order for all three plugins to work together, they have to be added to the same HTML element.</p>
+	<div class="wb-paginate wb-tagfilter provisional wb-filter"
+	data-wb-filter='{
+		"uiTemplate": "#filteredEventsList",
+		"section": ".events-list",
+		"selector": "article"
+	}'
+	data-wb-paginate='{
+		"itemsPerPage": 5,
+		"section": ".events-list",
+		"selector": "article",
+		"uiTarget": "#paginationCntnr"
+	}'>
+		<p class="h3 wb-fltr-info"><span data-nbitem></span> event(s)</p>
+		<div class="row">
+			<div class="col-sm-6 col-md-5 col-lg-4">
+				<details open>
+					<summary>
+						<h4>Filter events</h4>
```

Comment:
I suggest to remove the heading inside the summary because it is not well supported by UA and AT. The summary element do have an implicit button role and that invalidate the heading level semantic in the accessibility tree.

ref. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary